spyder/plugins/updatemanager/widgets/update.py

```diff
@@ -474,6 +520,59 @@ def start_install(self):
 
         subprocess.Popen(' '.join(cmd), shell=True)
 
+    def _start_updater(self):
+        """Start updater application."""
+        if self.get_conf('high_dpi_custom_scale_factor', section='main'):
+            scale_factors = self.get_conf(
+                'high_dpi_custom_scale_factors',
+                section='main'
+            )
+            scale_factor = float(scale_factors.split(";")[0])
+        else:
+            scale_factor = 1
+
+        info = {
+            "install_file": self.installer_path,
+            "conda_exec": find_conda(),
+            "env_path": sys.prefix,
+            "update_type": self.asset_info["update_type"],
+            "window_title": _("Spyder update"),
+            "scale_factor": scale_factor,
+            "initial_message": _(
+                "Updating Spyder, this will take a few minutes ..."
+            ),
+            "success_message": _(
+                "The update was succesful!<br>Spyder will be launched shortly"
+            ),
+            "failure_message": _("Unfortunately the update failed"),
+            "error_message": _("There was an error in the update process"),
+            "details_title": _("Show details"),
+            "font_family": self.get_conf(
+                "app_font/family", section="appearance"
+            ),
+            "font_size": int(
+                self.get_conf("app_font/size", section="appearance")
+            ),
+            "monospace_font_family": self.get_conf(
+                "monospace_app_font/family", section="appearance"
+            ),
+            "monospace_font_size": int(
+                self.get_conf("monospace_app_font/size", section="appearance")
+            ),
+            "interface_theme": self.get_conf("ui_theme", section="appearance"),
```

Comment:
```suggestion
            "interface_theme": "dark" if is_dark_interface() else "light",
```

The video you posted shows we're not setting the right value here. The problem is that the `ui_theme` option can have the value `automatic`, so we need a way that really tells if the interface is dark or light.